.github/workflows/rules.yml

```diff
@@ -1,7 +1,7 @@
 # A single CI script with github workflow
 name: Tests
 
-on: [push]
+on: [push, pull_request_target]
```

Comment:
Thanks, but I think it doesn't work. Could you please replace the `pull_request_target` with `pull_request`?